CISLibrary/drivers/HBridge/DRV8837.cpp

```diff
@@ -7,51 +7,38 @@
  * @param rev pin
  * @param sleep pin
  */
-DRV8837::DRV8837(PinName fwd, PinName rev, PinName nSleep)
-  : fwd(fwd), rev(rev), nSleep(nSleep)
-{
-  // set the pwm frequencies
-  this->fwd.period(1.0 / DRV8837_PWM_FREQ_HZ);
-  this->rev.period(1.0 / DRV8837_PWM_FREQ_HZ);
-
-  // disable initially
-  this->nSleep.write(1);
+DRV8837::DRV8837(PinName fwd, PinName rev, PinName nSleep) :
+  fwd(fwd), rev(rev), nSleep(nSleep, 0) {
+  // set the pwm periods
+  this->fwd.period_us(DRV8837_PWM_PERIOD_US);
```

Comment:
Add the period to the arguments of the constructor. What if we want one hbridge at 1ms and the other at 1ns